frontend/src/components/tracing/tracing.tsx

```diff
@@ -35,9 +35,13 @@ import { ClearButton } from "../buttons/clear-button";
 import { cn } from "@/utils/cn";
 import { PanelEmptyState } from "../editor/chrome/panels/empty-state";
 import { type ResolvedTheme, useTheme } from "@/theme/useTheme";
+import useResizeObserver from "use-resize-observer";
+
+const expandedRunsAtom = atom<Map<RunId, boolean>>(new Map<RunId, boolean>());
```

Comment:
this is helpful to keep state if we open/close the sidebar

Reply:
I intentionally wanted to see all runs expanded but this works too. Let's get feedback and see, easy to make this a config too.